realm/src/androidTest/java/io/realm/RealmQueryTest.java

```diff
@@ -459,4 +460,36 @@ public void testGeorgian() {
             assertEquals(sorted[i], stringOnlies2.get(i).getChars());
         }
     }
+
+    private RealmResults<CatOwner> results = null;
+    public void testModifyRealmListD() {
```

Comment:
Perhaps a small description saying that this test is nondeterministic and depends on the GC doing it's job at the wrong time.